packages/javascript-sdk/src/oauth2-client/index.ts

```diff
@@ -82,7 +85,8 @@ abstract class OAuth2Client {
    * New Name: getAuthCodeByIframe
    */
   public static async getAuthCodeByIframe(options: GetAuthorizationUrlOptions): Promise<string> {
-    const url = await this.createAuthorizeUrl(options);
+    const url = await this.createAuthorizeUrl({ ...options, prompt: 'none' });
```

Comment:
Given that we are setting prompt=none as a default, does it make sense to make this something the developer can specify? As I understand it, setting `prompt: 'login'` overrides this but if nothing is set then it's `prompt: 'none'`. Maybe I'm overthinking this but what do you think?

Reply:
Good question. This is how I look at it: since this is within the hidden iframe, I don't believe there is ever a reason to NOT use `prompt=none` as any prompt, aka UI, would be hidden and therefore unusable.

Reply:
Yes. Given this and my point about the value `consent` further up, perhaps we should rethink this. Maybe we should obscure this entirely from the user and always pass `prompt=none` unless the developer specifies they want to do centralized login using `login: 'redirect'`? That way we avoid introducing a "thing" that the developer has to think about.